### ISSUE TITLE
Gold cores using Performance governor instead of Schedutil

### ISSUE THREAD

**Issue by JacekJagosz** [sonyxperiadev/device-sony-lena#20]
Checking governor for all cores returns:
```bash
cat /sys/devices/system/cpu/cpu*/cpufreq/scaling_governor
schedutil
schedutil
schedutil
schedutil
schedutil
schedutil
performance
performance
```
Even though all cores should be set to `schedutil`: https://github.com/sonyxperiadev/device-sony-lena/blob/7345396811b2f096586e846ca2fc6a2630ebd87e/rootdir/vendor/etc/init/init.lena.pwr.rc#L76
So it seems for some reason this setting is not listened to.
I [reported this on their bugtracker already](https://forum.sailfishos.org/t/xperia-10-iii-uses-performance-governor-for-gold-cores-where-it-probably-shouldnt/12695), but I went through Sailfish OS config for Lena and it is not something it changes. So I really think it is not a problem with that OS, but with this setup.

**Comment by jerpelea** [comment]
2x 2 GHz – Kryo 560 Gold (Cortex-A77)
6x 1.7 GHz – Kryo 560 Silver (Cortex-A55)

**Comment by JacekJagosz** [comment]
> policy4 should be updated to policy6
Want me to create a PR, or will you do it?

**Comment by jerpelea** [comment]
please do

**Comment by jerpelea** [comment]
for another commit 
pdx213:/ # cat /sys/devices/system/cpu/cpufreq/policy0/scaling_available_frequencies
300000 576000 768000 1017600 1248000 1324800 1516800 1612800 1708800
pdx213:/ # cat /sys/devices/system/cpu/cpufreq/policy6/scaling_available_frequencies
300000 787200 979200 1036800 1248000 1401600 1555200 1766400 1900800 2073600

write /sys/devices/system/cpu/cpufreq/policy0/scaling_min_freq 576000
write /sys/devices/system/cpu/cpufreq/policy6/scaling_min_freq 787200

**Comment by robang74** [comment]
I took some times to evaluate the CPU governors and I found that the Xperia 10 III lacks of the `interactive` governor which is present in the Xperia 10 III kernel (this information is related to SFOS 4.5.0.19). However, `schedutil` seems better for those wish to use the smartphone also like a UNIX server which can be under duty also when the user do not interact with the display. On the other side, about power saving the `conservative` governor is considered obsolete and I found that `relaxed` would be a good candidate to add together with `chill` for those are interested on a smartphone-only device destination. 

https://github.com/robang74/redfishos/blob/b0db91b593a3a60ce2f481f8a3bda180c4ca5b8f/forum/knowhow/battery-recharge-and-power-saving.md

This link above reports my analysis and feedbacks about it are welcome.
